tools/alphafill_cofactor_survey/run_cofactor.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
```

Comment:
I think I see what you were trying to do here, but I don't think it's the right place - I don't see the reason we should have a complicated bash script that does this when you can more easily do the filtering in your `find_cofactor.py`

to be honest I think it would be a good idea to just remove this bash script and instead add to `find_cofactor.py` the ability to accept a list of UniProt IDs in a text file, as well as a list of cofactors to use a a filter, in this way you would have everything in a single script/step and not need to fiddle with bash. Bash in general can be error-prone and difficult to maintain, so when things get a bit more complicated such as this case it is better using a proper programming language instead